js/src/jsonrpc/interfaces/eth.js

```diff
@@ -180,10 +149,10 @@ The following options are possible for the \`defaultBlock\` parameter:
     desc: 'Makes a call or transaction, which won\'t be added to the blockchain and returns the used gas, which can be used for estimating the used gas.',
     params: [
       {
-        type: Object,
-        desc: 'See [eth_call](#eth_call) parameters, expect that all properties are optional.',
+        type: CallRequest,
```

Comment:
+1 Great, definately helps. When we have these we are actually moving closer to being able to just auto-detect the input/output formatters, i.e. we are one step closer to having the JS auto-generated.